Mathlib/Data/Fintype/Order.lean

```diff
@@ -129,20 +129,9 @@ noncomputable def toCompleteLinearOrder [LinearOrder α] [BoundedOrder α] : Com
 -- See note [reducible non-instances]
 /-- A finite boolean algebra is complete. -/
 @[reducible]
-noncomputable def toCompleteBooleanAlgebra [BooleanAlgebra α] : CompleteBooleanAlgebra α :=
-  -- Porting note: using `Fintype.toCompleteDistribLattice α` caused timeouts
-  { Fintype.toCompleteLattice α,
-    ‹BooleanAlgebra α› with
-    iInf_sup_le_sup_sInf := fun a s => by
-      convert (Finset.inf_sup_distrib_left s.toFinset id a).ge using 1
-      rw [Finset.inf_eq_iInf]
-      simp_rw [Set.mem_toFinset]
-      rfl
-    inf_sSup_le_iSup_inf := fun a s => by
-      convert (Finset.sup_inf_distrib_left s.toFinset id a).le using 1
-      rw [Finset.sup_eq_iSup]
-      simp_rw [Set.mem_toFinset]
-      rfl }
+noncomputable def toCompleteBooleanAlgebra [BooleanAlgebra α] : CompleteBooleanAlgebra α where
```

Comment:
Why did the proof obligation disappear?

Reply:
It was an exact copy of the proofs in `toCompleteDistribLattice`. This fix addresses the porting note.